campaignion_email_to_target/src/Message.php

```diff
@@ -31,11 +37,44 @@ class Message extends MessageTemplateInstance {
    */
   public function __construct($data) {
     $data += [
-      'from' => '[submission:values:first_name] [submission:values:last_name] <[submission:values:email]>',
-      'to' => '[email-to-target:contact.title] [email-to-target:contact.first_name] [email-to-target:contact.last_name] <[email-to-target:contact.email]>',
+      'fromName' => '[submission:values:first_name] [submission:values:last_name]',
+      'fromAddress' => '[submission:values:email]',
+      'toName' => '[email-to-target:contact.title] [email-to-target:contact.first_name] [email-to-target:contact.last_name]',
+      'toAddress' => '[email-to-target:contact.email]',
       'display' => '[email-to-target:contact.display_name]',
     ];
     parent::__construct($data);
+
+    // Save as property for compatibility.
+    $this->to = $this->to();
+    $this->from = $this->from();
```

Comment:
As far as I can see the only usage of these is in `src/Channel/Email.php` I think we can change it their instead of introding these for backwards compatibility.

Reply:
it's also used in `webform.php`

Reply:
Changed.